packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -329,42 +361,70 @@ const privateNameHandlerSpec: Handler<PrivateNameState & Receiver> & Receiver =
       const isAccessor = getId || setId;
 
       if (isStatic) {
-        // NOTE: This package has a peerDependency on @babel/core@^7.0.0, but these
-        // helpers have been introduced in @babel/helpers@7.1.0.
-        const helperName =
-          isMethod && !isAccessor
-            ? "classStaticPrivateMethodGet"
-            : "classStaticPrivateFieldSpecGet";
-
         // if there are any local variable shadowing classRef, unshadow it
         // see #12960
         unshadow(classRef.name, member.scope, innerBinding);
 
-        return t.callExpression(file.addHelper(helperName), [
-          this.receiver(member),
-          t.cloneNode(classRef),
-          t.cloneNode(id),
+        if (!process.env.BABEL_8_BREAKING && !newHelpers(file)) {
+          // NOTE: This package has a peerDependency on @babel/core@^7.0.0, but these
+          // helpers have been introduced in @babel/helpers@7.1.0.
+          const helperName =
+            isMethod && !isAccessor
```

Comment:
Aside: We should rename `isAccessor` to `isGetterOrSetter` now that we have accessor properties.